src/components/Comment/Comment.tsx

```diff
@@ -89,12 +90,43 @@ const timestampWrapperStyles = css`
   justify-content: center;
 `;
 
+const buttonStyles = css`
+  margin-top: 12px;
+  margin-bottom: 12px;
+  cursor: pointer;
+  background: #fff;
+  color: #c70000;
+  height: 24px;
+  font-size: 12px;
+  font-weight: bold;
+  text-overflow: ellipsis;
+  border-radius: 12px;
+
+  border: 1px solid ${palette.neutral[86]};
+  svg {
+    fill: ${palette.neutral[60]};
+  }
+
+  :hover {
+    border: 1px solid ${palette.neutral[60]};
+    svg {
+      fill: ${palette.neutral[46]};
+    }
+  }
+`;
+
 export const avatar = (avatarSize: number): string => css`
   border-radius: ${avatarSize + 10}px;
   width: ${avatarSize}px;
   height: ${avatarSize}px;
 `;
 
+const Plus = () => (
+  <svg width="14" height="14" viewBox="0 0 18 18">
+    <path d="M8.2 0h1.6l.4 7.8 7.8.4v1.6l-7.8.4-.4 7.8H8.2l-.4-7.8L0 9.8V8.2l7.8-.4.4-7.8z"></path>
+  </svg>
+);
+
```

Comment:
At some point these svgs could be centralised but it's nice to have them close to where they're being used at this early stage